### ISSUE TITLE
ModuleNotFoundError: No module named 'maintenance_mode'

### ISSUE THREAD

**Issue by geraldhe** [hholzgra/maposmatic-vagrant#6]
I think the python-module `django-maintenance-mode` or `django-maintenancemode-2` is missing?

```
    default: leaflet-search#2.9.7 www/static/bower/leaflet-search
    default:
    default: bootstrap#3.3.7 www/static/bower/bootstrap
    default: └── jquery#1.12.4
    default: Traceback (most recent call last):
    default:   File "manage.py", line 32, in <module>
    default:     execute_from_command_line(sys.argv)
    default:   File "/usr/lib/python3/dist-packages/django/core/management/__init__.py", line 364, in execute_from_command_line
    default:     utility.execute()
    default:   File "/usr/lib/python3/dist-packages/django/core/management/__init__.py", line 338, in execute
    default:     django.setup()
    default:   File "/usr/lib/python3/dist-packages/django/__init__.py", line 27, in setup
    default:     apps.populate(settings.INSTALLED_APPS)
    default:   File "/usr/lib/python3/dist-packages/django/apps/registry.py", line 85, in populate
    default:     app_config = AppConfig.create(entry)
    default:   File "/usr/lib/python3/dist-packages/django/apps/config.py", line 94, in create
    default:     module = import_module(entry)
    default:   File "/usr/lib/python3.6/importlib/__init__.py", line 126, in import_module
    default:     return _bootstrap._gcd_import(name[level:], package, level)
    default:   File "<frozen importlib._bootstrap>", line 994, in _gcd_import
    default:   File "<frozen importlib._bootstrap>", line 971, in _find_and_load
    default:   File "<frozen importlib._bootstrap>", line 953, in _find_and_load_unlocked
    default: ModuleNotFoundError: No module named 'maintenance_mode'
    default: Traceback (most recent call last):
    default:   File "manage.py", line 32, in <module>
    default:     execute_from_command_line(sys.argv)
    default:   File "/usr/lib/python3/dist-packages/django/core/management/__init__.py", line 364, in execute_from_command_line
    default:     utility.execute()
    default:   File "/usr/lib/python3/dist-packages/django/core/management/__init__.py", line 338, in execute
    default:     django.setup()
    default:   File "/usr/lib/python3/dist-packages/django/__init__.py", line 27, in setup
    default:     apps.populate(settings.INSTALLED_APPS)
    default:   File "/usr/lib/python3/dist-packages/django/apps/registry.py", line 85, in populate
    default:     app_config = AppConfig.create(entry)
    default:   File "/usr/lib/python3/dist-packages/django/apps/config.py", line 94, in create
    default:     module = import_module(entry)
    default:   File "/usr/lib/python3.6/importlib/__init__.py", line 126, in import_module
    default:     return _bootstrap._gcd_import(name[level:], package, level)
    default:   File "<frozen importlib._bootstrap>", line 994, in _gcd_import
    default:   File "<frozen importlib._bootstrap>", line 971, in _find_and_load
    default:   File "<frozen importlib._bootstrap>", line 953, in _find_and_load_unlocked
    default: ModuleNotFoundError: No module named 'maintenance_mode'
    default: Traceback (most recent call last):
    default:   File "manage.py", line 32, in <module>
    default:     execute_from_command_line(sys.argv)
    default:   File "/usr/lib/python3/dist-packages/django/core/management/__init__.py", line 364, in execute_from_command_line
    default:     utility.execute()
    default:   File "/usr/lib/python3/dist-packages/django/core/management/__init__.py", line 338, in execute
    default:     django.setup()
    default:   File "/usr/lib/python3/dist-packages/django/__init__.py", line 27, in setup
    default:     apps.populate(settings.INSTALLED_APPS)
    default:   File "/usr/lib/python3/dist-packages/django/apps/registry.py", line 85, in populate
    default:     app_config = AppConfig.create(entry)
    default:   File "/usr/lib/python3/dist-packages/django/apps/config.py", line 94, in create
    default:     module = import_module(entry)
    default:   File "/usr/lib/python3.6/importlib/__init__.py", line 126, in import_module
    default:     return _bootstrap._gcd_import(name[level:], package, level)
    default:   File "<frozen importlib._bootstrap>", line 994, in _gcd_import
    default:   File "<frozen importlib._bootstrap>", line 971, in _find_and_load
    default:   File "<frozen importlib._bootstrap>", line 953, in _find_and_load_unlocked
    default: ModuleNotFoundError: No module named 'maintenance_mode'
    default: CommandError: This script should be run from the Django Git checkout or your project or app tree, or with the settings module specified.
    default: Created symlink /etc/systemd/system/multi-user.target.wants/maposmatic-render.service → /lib/systemd/system/maposmatic-render.service.
```

**Comment by geraldhe** [comment]
editing the  `inc/install-packages.sh` and adding the following helped:

`python3-django-maintenancemode` to `apt-get install`
`django-maintenance-mode` and `django-maintenancemode-2` to `pip3 install`

but i don't know which of these three packages is necessary...

**Comment by hholzgra** [comment]
It turned out that pushes to this github project had failed for quite a while already due to me adding an SQL import file that exceeded the GitHub 100MB limit.

Unfortunately I didn't really pay attention to the error messages on push, mistaking them for mere warnings,
and when I finally noticed what was really going on it took a while to clean up the mess.

I had actually noticed the missing package install pretty quickly myself, but had added the big file right between breaking and fixing this.

Everything should be in a working state again now though, `django-maintenance-mode` was the missing pip3 package and is now getting installed properly by install.sh
